announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/user/prefs/AnnouncementUserNotificationPreferencesRegistrationImpl.java

```diff
@@ -30,4 +32,11 @@ public ResourceLoader getResourceLoader(String location) {
 		return new ResourceLoader(location);
 	}
 
+	@Setter
+	private ServerConfigurationService serverConfigurationService;
+
+	public void init() {
+		super.init();
+		setDefaultValue(serverConfigurationService.getString("prefs.default.noti.option.announcements"));
```

Comment:
String? it's an integer, isn't it ?

Reply:
Yeah, but it is handled as a String

**UserNotificationPreferencesRegistrationImpl.java**
> 	public void setDefaultValue(String defaultValue) {
		this.defaultValue = defaultValue;
	}